### PR TITLE
#55 allow users to change their vote

### PR DESCRIPTION
**Pull request by pmeads**
Add new buttons on idea/show.erb to give the user ability to change how they support an idea. They can decide to participate, withdraw participation, or withdraw support for the idea.

### CODE REVIEW

**Comment by pmeads** [comment]
This was the error:
/home/travis/build/universityofcalifornia/NeXt/db/schema.rb doesn't exist yet. Run `rake db:migrate` to create it, then try again. If you do not intend to use a database, you should instead alter 

I didn't get any errors in my env. The tests passed for me:

Finished in 1.29 seconds (files took 6.18 seconds to load)
23 examples, 0 failures, 2 pending

File: app/controllers/ideas/votes_controller.rb [hunk 1]
```diff
@@ -18,16 +18,53 @@ def create
...
+
+      if vote.participate && params[:participate]=="false"
+        page_alert_msg = "You have withrawn your participation from #{@idea.name}"
+      elsif ! vote.participate && params[:participate]=="true"
+        page_alert_msg = "Thank you for participating in #{@idea.name}!"
+      end
+
+      vote.participate = params[:participate]
+      vote.save
+
```
Comment (chabgood) [inline app/controllers/ideas/votes_controller.rb]: Lines 38-45 could be put in the model to clean up the controller.

page_alert_message = vote.save_participation(@idea, params[:participate])

**Comment by ebollens** [comment]
@pmeads please update this branch against master so it can be merged through.

**Comment by ebollens** [comment]
@pmeads please update this branch against master so it can be merged through.

**Pull request closed without merging**